website/docs/guides/best-practices/environment-setup/3-many-deployment-environments.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Many deployment environments"
+id: 3-many-deployment-environments
+description: Learn how to configure environments in dbt Cloud.
+displayText: "dbt Cloud environment best practices"
+hoverSnippet: Learn how to configure environments in dbt Cloud.
+---
+
+## Many deployment environments
+
+### **What this looks like**
+
+1. You have a **single *development* environment** where dbt users can access the dbt Cloud IDE and make changes to their code. However, you’ll want to update the **[custom branch settings](faqs/Environments/custom-branch-settings)** to ensure that developers create feature branches off of the a non-production branch. For this example, we’ll refer to this as the `qa` branch.
+2. You have a **QA deployment environment**, running scheduled jobs from the `qa` branch that deploys your dbt project to a pre-production warehouse location.
+3. You have a **Production deployment environment,** running scheduled jobs from the `main` branch that deploys your dbt project to your production warehouse location.
+4. You have **multiple Slim CI jobs** (one in each deployment environment) to ensure changes to each branch are tested.  
+
+![Table of basic setup for many deployment environment](/img/guides/best-practices/environment-setup/many-deployments-table.png)
+
+### git workflow
+
+![git flow diagram for many deployment environments](/img/guides/best-practices/environment-setup/many-branch-git.png)
+
+1. In the dbt Cloud IDE, developers work on feature branches, **created from the `qa` branch** (`feature_a`, `feature_b`, `feature_c` above).
+2. When code is ready, developer opens a PR to merge feature branch into `qa`.
+3. The **first Slim CI Job** automatically kicks off to test the changes introduced in the PR. This job will *defer to a regularly-scheduled job in the QA environment.*
+4. When **Slim CI Job is successful** and team is ready to deploy changes, the **PR is merged into `qa`.**
+5. Scheduled jobs run in QA environment, running on `qa` branch to ensure the new changes work as indended.
+6. When **all feature branches** for a given release (e.g. sprint) have been **successfully merged** to `qa` and are **running without error** in the pre-production environment, a team member opens a **PR to merge `qa` → `main`.**
+7. The **second Slim CI Job** automatically kicks off to test changes in PR. This job will *defer to a regularly-scheduled job in the Production environment.*
+8. When **second Slim CI Job** is successful and team is ready to deploy changes, the **PR is merged into `main`**.
+9. Monitor scheduled jobs in Prod environment that are running on `main` branch. Voila! All changes are released and ready for your stakeholders!
+
+:::info
+💡 Considering a different branching strategy that involves cherry picking? [Maybe reconsider!](blog/the-case-against-git-cherry-picking)
```

Comment:
the red underline makes me think something is spelled wrong

Reply:
i am not sure where that is coming from!